src/pages/Profile/Profile.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from 'react';
+import { ConfigProvider, Tabs } from 'antd';
+import { instanceOf } from 'prop-types';
+import { useNavigate, useParams } from 'react-router-dom';
+
+import ProfileGeneralInfo from '../../components/profile/ProfileGeneralInfo';
+import ProfileDUICrimHistory from '../../components/profile/ProfileDUICrimHistory';
+import ProfileRolesSkills from '../../components/profile/ProfileRolesSkills';
+import ProfileAvailability from '../../components/profile/ProfileAvailability';
+import WaiversGrid from '../../components/waivers/WaiversGrid';
+
+import { Cookies, cookieKeys, withCookies } from '../../util/cookie_utils';
+import AUTH_ROLES from '../../util/auth_config';
+import { AFCBackend } from '../../util/auth_utils';
+
+import styles from './Profile.module.css';
+
+const { TabPane } = Tabs;
+const Profile = ({ cookies }) => {
+  const { userId } = useParams();
+  const navigate = useNavigate();
+
+  const [user, setUser] = useState(null);
+  // TODO Replace waivers with actual set of waivers
+  const [waivers, setWaivers] = useState([
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'setWaivers' is assigned a value but never used.